src/pages/dashboard/InstitutionFiltering.tsx

```diff
@@ -66,26 +109,30 @@ const InstitutionFiltering: FC<InstitutionFilteringProps> = ({ queryObject, setQ
       <FormLabel>
         <Typography variant="h3">{t('dashboard.institutions')}</Typography>{' '}
       </FormLabel>
-      <FormGroup>
-        {institutionCheckedList.map((institution, index) => (
-          <FormControlLabel
-            data-testid={`institution-filtering-checkbox-label-${institution.name.toLowerCase()}`}
-            key={index}
-            control={
-              <Checkbox
-                data-testid={`institution-filtering-checkbox-${institution.name.toLowerCase()}`}
-                color="default"
-                checked={institution.isSelected}
-                name={institution.name}
-              />
-            }
-            label={t(`institutions.${institution.name}`)}
-            onChange={(event) => {
-              changeSelected(index, event);
-            }}
-          />
-        ))}
-      </FormGroup>
+      {isLoading && <CircularProgress />}
```

Comment:
`{isloading ? <CircularProgress /> : <the rest>}`

Reply:
Fikset